rb/Gemfile.lock

```diff
@@ -71,4 +70,4 @@ DEPENDENCIES
   yard (~> 0.9.11)
 
 BUNDLED WITH
-   1.16.3
```

Comment:
Let's not bump bundler version since we ship 1.16.3 by default. Another approach would be to bump it in `third_party/rb'?